aiida/engine/runners.py

```diff
@@ -150,7 +150,10 @@ def close(self):
         """Close the runner by stopping the loop."""
         assert not self._closed
         self.stop()
-        reset_event_loop_policy()
+        try:
+            reset_event_loop_policy()
+        except AttributeError:
```

Comment:
I see no reason to pass this exception, shouldn't let it raised? And why the `AttributeError` capture here?

Reply:
Good point. I had to add it there at some point to get the test to pass but the failure was because I wasn't overriding the `event_loop` fixture of `pytest-asyncio`. Now it is no longer needed.